src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,31 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+
+        Direction robotDirection;
+
+        if (robot.getX() != toX) {
+
+            robotDirection = (robot.getX() < toX) ? Direction.RIGHT : Direction.LEFT;
+
+            while (!robotDirection.equals(robot.getDirection())) {
+                robot.turnLeft();
+            }
+            while (!(robot.getX() == toX)) {
+                robot.stepForward();
+            }
+        }
+
+        if (robot.getY() != toY) {
+
```

Comment:
we don't need all those empty lines, they don't improve code readability, but make it longer, let's remove it